java/src/jmri/jmrit/display/layoutEditor/LayoutEditorBundle.properties

```diff
@@ -556,6 +556,8 @@ Error16 = Error - Cannot attach Memory to Block because no memory is defined for
 Error17 = Error - Reporter manager not available. Cannot add Reporter Icon.
 Error18 = Error - Could not provide Reporter "{0}". Cannot add Reporter Icon.
 Error19 = Error - Missing Name for Sensor "{0}". Please fill in all fields.
+Error20 = Error - One slip turnout is empty\nBoth turnouts must be empty or have two different turnouts assigned.
```

Comment:
No need to change. And I might be too picky.
But maybe add a dot after `One slip turnout is empty`